applications/accounts/web/src/styles.css

```
@@ -1,5 +1,7 @@
 @layer theme, base, components, utilities;
 
+@reference '@motech-development/breeze-ui/theme.css';
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@reference`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)